preprocess.py

```diff
@@ -10,23 +10,31 @@ def load_bin_vec(fname, vocab):
     Loads 300x1 word vecs from Google (Mikolov) word2vec
     """
     word_vecs = {}
+    print(len(vocab))
     with open(fname, "rb") as f:
         header = f.readline()
-        vocab_size, layer1_size = map(int, header.split())
+        vocab_size, layer1_size = list(map(int, header.split()))
         binary_len = np.dtype('float32').itemsize * layer1_size
-        for line in xrange(vocab_size):
+        i = 0
+        for line in range(vocab_size):
+            # print(i+1)
+            # i+=1
             word = []
             while True:
-                ch = f.read(1)
+                try:
+                  ch = f.read(1).decode("utf-8")
```

Comment:
This line was not working without `.decode("utf-8") for certain lines in `GoogleNews-vector-bin`. Needed `try-except` because of some bad end of line characters in  `GoogleNews-vector-bin`.